tests/mocha/navigation_test.js

```diff
@@ -50,8 +57,8 @@ suite('Navigation', function() {
     });
 
     teardown(function() {
+      workspaceTeardown.call(this, this.workspace);
```

Comment:
Could this also go in the parent teardown function?

Reply:
Can't be moved, parent teardown doesn't have access to `this.workspace`. Could be moved if `this.workspace` was re-defined in outer scope, but there's a lot of other similar instances and I'm not yet convinced that's the best plan. See #4080 for more explanation (if I do move things, I'd probably do it as part of addressing that bug).